.ci/test_wrapper.sh

```diff
@@ -34,5 +34,13 @@ then
 else
     REPO_DIR="/root/keylime"
 fi
+
+# Remove /etc/keylime.conf because there might a old one distributed with the container.
+if [ -f "/etc/keylime.conf" ]
+then
+    echo "Removing old /etc/keylime.conf"
+    rm /etc/keylime.conf
```

Comment:
Instead of removing it, what do you think about copying it to say `/etc/keylime.conf.bak` or something. This is just for running the tests, but sometimes as a developer I run the ci wrapper just to make sure everything is working, and I'd hate to lose anything I'd done to the keylime.conf just in case.

Reply:
The config is now moved to `/etc/keylime.conf.orig` instead of deleting it.